src/Common/MoistThermodynamics/MoistThermodynamics.jl

```diff
@@ -840,7 +840,7 @@ function liquid_ice_pottemp_given_pressure(T::FT, p::FT,
   q::PhasePartition{FT}=q_pt_0(FT)) where {FT<:Real}
     # liquid-ice potential temperature, approximating latent heats
     # of phase transitions as constants
-    return dry_pottemp_given_pressure(T, p, q) * (1 - latent_heat_liq_ice(q)/(cp_m(q)*T))
+    return dry_pottemp_given_pressure(T, p, q) * (1 - latent_heat_liq_ice(q)/(FT(cp_d)*T))
```

Comment:
To make this consistent, you will need to compute `dry_pottemp_given_pressure(T, p, q)` with zero specific humidity `q`. You will also make analogous changes in all other computations involving the liquid-ice potential temperature.

Reply:
If we use cp_m here, I wonder if we should use LH_v and LH_s (temperature dependent) instead of LH_v0 and LH_s0 in latent_heat_liq_ice to make it consistent. (This will probably affect the relationship between q_t and q_l the same way as changing cp_m to cp_d).

Reply:
Good point—just makes it harder to invert for temperature, which is why we don’t have the temperature dependent latent heats.

Reply:
I updated the design docs with constant thermodynamic coefficients for the liquid-ice modification of the potential temperature. If you want to change things, please do as stated there. I left the dry potential temperature with \kappa = R_m/c_{pm}. For the inversion given density, this makes an additional approximation necessary, but it still holds to first order. However, none of this will make much of a difference for DYCOMS--the problems there lie elsewhere. Another option is simply to leave `c_{pm}`, `c_{vm}` etc. as we had it. Either way is fine and will not matter much for what we are interested in.

Reply:
Ok, if this is not a high priority then I'll leave things as is for now, but I'll keep the PR open for reference.